### DISCUSSION TITLE
Hash로 풀 수 있는 문제를 올려주세요 !

### DISCUSSION THREAD

**Discussion by tony9402** [tony9402/baekjoon#141] [문제 추가/수정 요청]
댓글로 남겨주시면 감사하겠습니다.

**형식**
문제번호 : 1000
문제제목 : A+B
문제링크 : https://www.acmicpc.net/problem/1000

**Comment by gusdn3477** [comment]
문제번호 : 1920
문제제목 : 수 찾기
문제링크 : https://www.acmicpc.net/problem/1920

문제번호 : 2776
문제제목 : 암기왕
문제링크 : https://www.acmicpc.net/problem/2776

**Comment by cdog-gh** [comment]
문제번호 : 21276
문제제목 : 계보 복원가 호석
문제링크 : https://www.acmicpc.net/problem/21276

문제번호 : 7785
문제제목 : 회사에 있는 사람
문제링크 : https://www.acmicpc.net/problem/7785

**Reply by cdog-gh** [reply]
int를 key로 주는 경우, **unordered_map이나 unordered_set은 주의 필요**합니다.
